2021-05-28 04:41:14,110 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting env: {}
2021-05-28 04:41:14,110 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-28 04:41:14,110 INFO    MainThread:29752 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_044114-3tgrtbva/logs/debug.log
2021-05-28 04:41:14,110 INFO    MainThread:29752 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_044114-3tgrtbva/logs/debug-internal.log
2021-05-28 04:41:14,110 INFO    MainThread:29752 [wandb_init.py:init():369] calling init triggers
2021-05-28 04:41:14,110 INFO    MainThread:29752 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 192, 'batch_size': 32, 'init_lr': 0.0014685305800254855}
2021-05-28 04:41:14,110 INFO    MainThread:29752 [wandb_init.py:init():418] starting backend
2021-05-28 04:41:14,110 INFO    MainThread:29752 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-28 04:41:14,120 INFO    MainThread:29752 [backend.py:ensure_launched():132] starting backend process...
2021-05-28 04:41:14,127 INFO    MainThread:29752 [backend.py:ensure_launched():136] started backend process with pid: 29761
2021-05-28 04:41:14,139 INFO    MainThread:29752 [wandb_init.py:init():423] backend started and connected
2021-05-28 04:41:14,140 INFO    MainThread:29752 [wandb_init.py:init():465] updated telemetry
2021-05-28 04:41:14,140 INFO    MainThread:29752 [wandb_init.py:init():484] communicating current version
2021-05-28 04:41:14,887 INFO    MainThread:29752 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-28 04:41:14,887 INFO    MainThread:29752 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-28 04:41:15,110 INFO    MainThread:29752 [wandb_init.py:init():522] starting run threads in backend
2021-05-28 04:41:15,144 INFO    MainThread:29752 [wandb_run.py:_console_start():1565] atexit reg
2021-05-28 04:41:15,145 INFO    MainThread:29752 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-28 04:41:15,145 INFO    MainThread:29752 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-28 04:41:15,147 INFO    MainThread:29752 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-28 04:41:15,147 INFO    MainThread:29752 [wandb_init.py:init():546] run started, returning control to user process
2021-05-28 04:52:10,278 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting env: {}
2021-05-28 04:52:10,278 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-28 04:52:10,278 INFO    MainThread:29752 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_045210-3q0ojjik/logs/debug.log
2021-05-28 04:52:10,278 INFO    MainThread:29752 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_045210-3q0ojjik/logs/debug-internal.log
2021-05-28 04:52:10,278 INFO    MainThread:29752 [wandb_init.py:init():369] calling init triggers
2021-05-28 04:52:10,278 INFO    MainThread:29752 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 288, 'batch_size': 8, 'init_lr': 0.002023075716417512}
2021-05-28 04:52:10,279 INFO    MainThread:29752 [wandb_init.py:init():389] re-initializing run, found existing run on stack: 3tgrtbva
2021-05-28 04:52:10,279 INFO    MainThread:29752 [wandb_run.py:finish():1208] finishing run seokjin/optuna/3tgrtbva
